tools/Azure.Mcp.Tools.AzureBestPractices/src/AzureBestPracticesSetup.cs

```diff
@@ -10,7 +10,7 @@ namespace Azure.Mcp.Tools.AzureBestPractices;
 
 public class AzureBestPracticesSetup : IAreaSetup
 {
-    public string Name => "get_bestpractices";
+    public string Name => "bestpractices";
```

Comment:
We need to revert this change back, before checking this in.

Reply:
@fanyang-mono I thought you said it was fine to change the name. I'll need to update the tooling to make it a special case if we want to keep `get_bestpractices`, as it produces a tool called `azmcp get gestpractices get`. Why can't this tool follow the convention all others do?